### DISCUSSION TITLE
[Bug] Possible conflict between mermaid2 plugin and integrated mermaid.js in mkdocs-materials

### DISCUSSION THREAD

**Discussion by h7b** [ObsidianPublisher/obsidian-github-publisher#125] [Q&A — Template] [answered]
Hi

Since v8.2, Material for MkDocs integrates with [Mermaid.js](https://mermaid-js.github.io/mermaid/) natively [^1], I think we don't need to include the [mermaid2](https://github.com/fralau/mkdocs-mermaid2-plugin) plugin.

[^1]: https://squidfunk.github.io/mkdocs-material/reference/diagrams/

When using both of them in the `mkdocs.yml`, i found that the diagram is not displayed correctly.

This is the mermaid code that I used
```code-type-mermaid
erDiagram
    BANKS ||--o{ PRODUCTS : has
    BANKS {
        int bank_id
        string bank_name
    }
    PRODUCTS {
        int product_id
        int loan_amount
        float interest_rate
        string accepted_risk_level
        int bank_id
        datetime created_date
    }
    CUSTOMERS ||--o{ LEADS : is
        CUSTOMERS {
        int customer_id
        string customer_name
        int customer_age
        string estimated_risk_level
        string source
        datetime created_date
    }
    PRODUCTS ||--o{ LEADS : applied_by
    LEADS {
        int customer_id
        int product_id
        datetime apply_date
    }
```

Here is the screencap of published page when both plugin mermaid2 and integrated mermaid.js were enabled
![Screenshot 2023-03-26 040206](https://user-images.githubusercontent.com/86811584/227751600-dbdc177d-065f-4695-84b7-dbd685f406bc.jpg)

Next is the screencap of published page where the plugin mermaid2 is disabled
![Screenshot 2023-03-26 041723](https://user-images.githubusercontent.com/86811584/227751648-e53fb411-359c-44ae-80d2-0dddf8bcbc2f.jpg)

After re-verification, hope you have time to update the template

Thanks

**Comment by Mara-Li** [comment]
I was pretty sure I forgot something about mermaid.
Thanks I will update it asap.

**Comment by Mara-Li** [comment, marked as answer]
cf #126
